src/main/antlr4/imports/column_definitions.g4

```diff
@@ -67,7 +69,7 @@ column_options:
 	  nullability
 	| default_value
 	| primary_key
-	| ON UPDATE ( CURRENT_TIMESTAMP | now_function )
+	| ON UPDATE ( CURRENT_TIMESTAMP length? | now_function )
```

Comment:
http://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_now